css/styles.css

```
@@ -0,0 +1,31 @@
+@font-face {
+    font-family: "Rubik";
+    font-style: normal;
+    font-weight: 400;
+    src: url("fonts/rubik/rubik-400.woff2") format("woff2");
```

```suggestion
    src: url("../fonts/rubik/rubik-400.woff2") format("woff2");
```
Это относительный путь, а это значит, что он относительно того файла, в котором этот путь указан. Про абсолютные и относительные пути [здесь](https://up.htmlacademy.ru/htmlcss/32/book/2.3).